### PR TITLE
Create update.json for KaTeX

### PR DESCRIPTION
**Pull request by as-com**
I hope this works...

### CODE REVIEW

File: files/katex/update.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "bower",
+  "name": "katex",
+  "repo": "Khan/KaTeX",
+  "files": {
+    "basePath": "bower_components/katex/dist",
+    "exclude": ["README.md"]
```
Comment (megawac) [inline files/katex/update.json]: I don't think this is right:

https://github.com/Khan/KaTeX/tree/v0.4.0

I think `/dist` would work though?

Comment (as-com) [inline files/katex/update.json]: @megawac Ok, fixed that

File: files/katex/update.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "bower",
```
Comment (megawac) [inline files/katex/update.json]: Whoops, I missed you were using `bower`. Can you let me know if its good to merge @as-com?

Comment (as-com) [inline files/katex/update.json]: @megawac So do I need to add `/bower_components/katex`?

Comment (megawac) [inline files/katex/update.json]: Or change to github. I'm not sure what the `bower` file structure looks like. I haven't used KaTex myself so I don't know what files should be expected

**Comment by as-com** [comment]
Ok, I think it's ready to merge. I squashed the commits as well.

**Comment by megawac** [comment]
Awesome thanks!

**Pull request merged**